CHANGELOG.md

```diff
@@ -11,7 +11,7 @@ project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Added
 
-- Test against minimum versions specified in `versions.tf` (by @dpiddockcmp)
```

Comment:
Why are you removing change log entry?

Reply:
Sorry, this was a mistake.
I added back the entry.